apps/src/componentLibrary/StylizedBaseDialog.jsx

```diff
@@ -123,7 +124,9 @@ export default function StylizedBaseDialog(props) {
 
 StylizedBaseDialog.propTypes = {
   title: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
-  body: PropTypes.oneOfType([PropTypes.element, PropTypes.string]).isRequired,
+  body: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
+  // Alternative to providing body prop
+  children: PropTypes.node,
```

Comment:
I'd recommend leaving `children` unspecified so we don't overwrite a lot of the goodness already provided by React: https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx

Reply:
Our linter wouldn't let me do that I think?

Reply:
Yeah, confirmed linter required it